src/etc/inc/gwlb.inc

```diff
@@ -699,15 +699,28 @@ function return_gateways_array($disabled = false, $localhost = false, $inactive
 			default:
 				$tunnelif = substr($ifcfg['if'], 0, 3);
 				if (substr($ifcfg['if'], 0, 4) == "ovpn") {
-					// if current iface is an ovpn server endpoint then check its type, skip tap only
-					if (substr($ifcfg['if'], 4, 1) == 's') {
-						$ovpnid = substr($ifcfg['if'], 5);
-						if (is_array($config['openvpn']['openvpn-server'])) {
-							foreach ($config['openvpn']['openvpn-server'] as & $ovpnserverconf) {
-								if ($ovpnserverconf['vpnid'] == $ovpnid) {
-									if ($ovpnserverconf['dev_mode'] == "tap") {
-										continue 3;
-									}
```

Comment:
Above code is only dealing with `openvpn-server` items, and below it's also considering `openvpn-client`. Could you please elaborate a bit more why are you changing that behavior?